bot/events/removeRoleReactionHandler.js

```diff
@@ -0,0 +1,28 @@
+const source = require('rfr');
+
+const { changeRole, findRole, Mode } = source('bot/roles/roles');
+const { getMember } = source('bot/utils/util');
+const {checkForReactionPartial, getRoleForReaction} = source('bot/reactions/botReactions');
+
+function removeRoleReactionHandler(member, roleName) {
+    const role = findRole(member, roleName);
+    changeRole(member, role, Mode.REMOVE);
+}
+
+function removeReactionHandler(reaction, user) {
+
+    if (reaction.message.guild === null || user.bot) return;
+
+    checkForReactionPartial(reaction);
+    getRoleForReaction(reaction)
+        .then((mapping) => {
+            if (mapping === null) return;
```

Comment:
Consider ternary

Reply:
Or null coalescing operator

Reply:
these does not need to return anything so no thanks